src/libraries/CallLib.sol

```diff
@@ -19,14 +19,23 @@ library CallLib {
     bytes data;
   }
 
-  /* ============ Custom Errors ============ */
+  /* ============ Events ============ */
+
+  /**
+   * @notice Emitted if a call to a target contract fails.
+   * @param callIndex Index of the call
+   * @param errorData Error data returned by the call
+   */
+  event CallFailure(uint256 callIndex, bytes errorData);
 
   /**
-   * @notice Custom error emitted if a call to a target contract fails.
-   * @param callIndex Index of the failed call
-   * @param errorData Error data returned by the failed call
+   * @notice Emitted if a call to a target contract succeeds.
+   * @param callIndex Index of the call
+   * @param successData Error data returned by the call
    */
-  error CallFailure(uint256 callIndex, bytes errorData);
+  event CallSuccess(uint256 callIndex, bytes successData);
```

Comment:
Likewise for this guy

Reply:
Done: https://github.com/pooltogether/ERC5164/pull/15/commits/2087493cd78cd53a7cf801e693437638e4803bae